spring-pulsar-docs/src/main/asciidoc/application-properties.adoc

```diff
@@ -15,3 +15,5 @@ include::application-properties/pulsar-client.adoc[]
 include::application-properties/pulsar-producer.adoc[]
 
 include::application-properties/pulsar-consumer.adoc[]
+
+include::application-properties/pulsar-administration.adoc[]
```

Comment:
Here I'm not sure how these docs are generated, I just added the stub for consistency